test/JDBC/upgrade/14_6/SYSTEM_USER_dep-vu-cleanup.mix

```diff
@@ -0,0 +1,5 @@
+-- tsql
```

Comment:
These files are in the wrong place. Let's move it to the correct location or remove them. It doesn't need to be with this PR, but let's try not to forget this. Same files are in 14_5 and latest folders as well.